SeriesMaster-Frontend/App/Screens/DiscoverScreen.js

```diff
@@ -9,6 +9,16 @@ import CardImage from "../components/CardImage";
 import PropTypes from "prop-types";
 import { getShows } from "../api/getShows";
 import useDebounce from "../hooks/useDebounce";
+import registerForPushNotifications from "../notifications/register";
+import * as Notifications from "expo-notifications";
+
+Notifications.setNotificationHandler({
+  handleNotification: async () => ({
```

Comment:
Why is this `async`?

Reply:
I was following the instructions on the expo documentation. But, yes it seems it's not really necessary in this case.